ui-tests/pom.xml

```diff
@@ -29,6 +29,13 @@
             <artifactId>uitestframework</artifactId>
             <version>${uitestframeworkVersion}</version>
         </dependency>
+        
```

Comment:
You're absolutely correct! Putting it in this one was a quick fix, since I wasn't sure if everything would pass tests without it. I'll go add that to the uitestframework pom. I'll take it out of here as well